Doppler.HtmlEditorApi.Test/IntegrationTest/accounts/accountName/campaings/content/GetCampaignTest.cs

```diff
@@ -0,0 +1,94 @@
+using Doppler.HtmlEditorApi;
+using Doppler.HtmlEditorApi.Storage.DapperProvider;
+using Doppler.HtmlEditorApi.Test.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Moq;
+using System.Text.Json;
+using System.Net;
+using TUD = Doppler.HtmlEditorApi.Test.Utils.TestUsersData;
+using Xunit;
+using Xunit.Abstractions;
+using System.Threading.Tasks;
+
+//namespace Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
```

Comment:
I think that we should define a namespace for this class, maybe simply `Doppler.HtmlEditorApi`

Reply:
```suggestion
namespace Doppler.HtmlEditorApi;

```